src/DynamoCore/Utilities/UpdateValueParams.cs

```diff
@@ -0,0 +1,18 @@
+﻿using ProtoCore.Namespace;
```

Comment:
Please move this class alongside `ModelBase` class because that's where `UpdateValue` method is introduced, don't see the need of having a new class (hey, two files down).